apps/cookbook/src/app/showcase/toggle-button-showcase/toggle-button-showcase.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+
+import { ShowcaseProperty } from '../../shared/showcase-properties/showcase-property';
+declare var require: any;
+
+@Component({
+  templateUrl: './toggle-button-showcase.component.html',
+})
+export class ToggleButtonShowcaseComponent {
+  exampleHtml: string = require('!raw-loader!../../examples/toggle-button-example/toggle-button-example.component.html')
```

Comment:
We expose the template through a config object now - e.g. see `apps\cookbook\src\app\examples\segmented-control-example\default\default.ts`